README.md

```diff
@@ -40,9 +50,13 @@ end
 ```
 
 - What is mystery2(123)?
+-6
 - What is mystery2(9005)?
+- 14
 - What is mystery2(-123)?
+- -123
 - _Added Fun: How could we make `mystery2(-123)` work the way we might expect it to work instead of the way it does?_
+ - Depends on if we want -123 to be -1 + 2 + 3 or - (1 + 2 + 3). Either case we need to add a condition that checks to see if the integer passed is negative - if true, than decided if we should take the absolute value - run the method and at the end make it negative again
```

Comment:
Excellent observation!! I'd want the latter, but wanted ambiguity to see what people would consider and assume.